.mergify.yml

```diff
@@ -24,29 +24,29 @@ pull_request_rules:
           {{ title }} (#{{ number }})
           {{ body }}
 
-  - name: Backport patches to the release/v10.0.x branch
+  - name: Backport patches to the release/v11.x branch
```

Comment:
Same as above, maybe we should put a comment that this is supposed to be the last 3 releases included here